### PR TITLE
doc/radosgw: add push_endpoint for rabbitmq

### PR DESCRIPTION
**Pull request by zdover23**
This commit directs users of rabbitmq to use "push_endpoint" (with an underscore) instead of "push-endpoint" (with a hy- phen). This commit adds a note that contains such a direct- ive. It does not alter the examples already present in the text.

Signed-off-by: Zac Dover <zac.dover@gmail.com>

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

File: doc/radosgw/notifications.rst [hunk 1]
```diff
@@ -96,6 +96,7 @@ Topics
 .. note::
 
     In all topic actions, the parameters are URL encoded, and sent in the message body using ``application/x-www-form-urlencoded`` content type
```
Comment (colemitchell) [inline doc/radosgw/notifications.rst:98]: @zdover23 Orthogonal to this PR, I would put a _the_ after _using_ and omit the second comma. Also, the use of periods at the end of these sentences is inconsistent.

**Review by colemitchell — APPROVED** [review]

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/48487 - Quincy backport
https://github.com/ceph/ceph/pull/48488 - Pacific backport

File: doc/radosgw/notifications.rst (hunk 1 above)
Comment (zdover23) [reply doc/radosgw/notifications.rst:98]: @colemitchell: I'll make a PR tomorrow that addresses these usage and punctuation-related deficiencies.

Comment (zdover23) [reply doc/radosgw/notifications.rst:98]: @colemitchell: https://github.com/ceph/ceph/pull/48493

**Comment by yuvalif** [comment]
@zdover23 could you please explain the purpose of this change?
the code here: https://github.com/ceph/ceph/blob/main/src/rgw/rgw_rest_pubsub.cc#L89
is using "push-endpoint" regardless of topic type (for amqp, kafka and http)

**Comment by zdover23** [comment]
> @zdover23 could you please explain the purpose of this change? the code here: https://github.com/ceph/ceph/blob/main/src/rgw/rgw_rest_pubsub.cc#L89 is using "push-endpoint" regardless of topic type (for amqp, kafka and http)

@yuvalif: That's a good question. I'll make a pull request now, restoring the hyphen. I'll make you the reviewer, and we won't merge until you're satisfied.

**Comment by zdover23** [comment]
@yuvalif: Check it out: https://github.com/ceph/ceph/pull/51299

**Comment by yuvalif** [comment]
> @yuvalif: Check it out: #51299

thanks!
BTW, looks like this change was made in quincy and pacific as well

**Pull request merged**